### PR TITLE
feature: add `_generate_works_json` method

### PR DESCRIPTION
**Pull request by yurijmikhalevich**
## What does this PR do?

This PR adds the internal `_generate_works_json` helper.

#### To run, do

```python
python -c "import lightning.app.runners.cloud as c;c._generate_works_json_web('/Users/yurij/code/lightning/examples/app_v0/app.py')"
```

Fixes #\<issue_number>

## Before submitting

- [x] Was this **discussed/approved** via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/Lightning-AI/lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your **PR does only one thing**, instead of bundling different changes together?
- [ ] Did you make sure to **update the documentation** with your changes? (if necessary)
- [ ] Did you write any **new necessary tests**? (not for typos and docs)
- [ ] Did you verify new and **existing tests pass** locally with your changes?
- [ ] Did you list all the **breaking changes** introduced by this pull request?
- [ ] Did you **update the CHANGELOG**? (not for typos, docs, test updates, or minor internal changes/refactors)

## PR review

Anyone in the community is welcome to review the PR.
Before you start reviewing, make sure you have read the [review guidelines](https://github.com/Lightning-AI/lightning/wiki/Review-guidelines). In short, see the following bullet-list:

- [x] Is this pull request ready for review? (if not, please submit in draft mode)
- [ ] Check that all items from **Before submitting** are resolved
- [ ] Make sure the title is self-explanatory and the description concisely explains the PR
- [ ] Add labels and milestones (and optionally projects) to the PR so it can be classified

## Did you have fun?

Make sure you had fun coding 🙃

cc @borda

### CODE REVIEW

File: src/lightning_app/runners/cloud.py [hunk 1]
```diff
@@ -70,6 +71,115 @@
...
+            drives=drive_specs,
+            user_requested_compute_config=user_compute_config,
+            network_config=[V1NetworkConfig(name=random_name, port=work.port)],
+        )
+        works.append(V1Work(name=work.name, spec=work_spec))
+
+    return works
+
+
+def _to_web_ready_dict(swagger_object):
```
Comment (tchaton) [inline src/lightning_app/runners/cloud.py:153]: Can you add some tests ?

Comment (ethanwharris) [reply src/lightning_app/runners/cloud.py]: Done :smiley:

File: src/lightning_app/runners/cloud.py [hunk 2]
```diff
@@ -70,6 +71,120 @@
...
+            if value is not None and value != {}:
+                key = attribute_map[key] if map_attributes else key
+                result[key] = value
+        return result
+    elif isinstance(swagger_object, list):
+        return [_to_clean_dict(x, map_attributes) for x in swagger_object]
+    elif isinstance(swagger_object, dict):
+        return {key: _to_clean_dict(value, map_attributes) for key, value in swagger_object.items()}
+    else:
+        return swagger_object
```
Comment (Borda) [inline src/lightning_app/runners/cloud.py:170]:
```suggestion
    return swagger_object
```

**Review by Borda — APPROVED** [review]

**Review by tchaton — APPROVED** [review]
LGTM !

**Pull request merged**